in_toto/gpg/constants.py

```diff
@@ -30,7 +29,8 @@
 FULLY_SUPPORTED_MIN_VERSION = "2.1.0"
 
 try:
-  subprocess.check_call(shlex.split(GPG_VERSION_COMMAND))
+  process.run(GPG_VERSION_COMMAND, stdout=process.DEVNULL,
+    stderr=process.DEVNULL)
```

Comment:
Just realized that this of course fixes the issue I mentioned yesterday that there is gpg noise on stdout if you run in-toto from the develop branch. 👍

Reply:
That noise is still there (see the unit test), I need someone to help me look at where this is coming from, I don't have the headspace for this now...

Reply:
Oh yeah, you're right, but it was only in the unit tests (see [`runtests.py`](https://github.com/in-toto/in-toto/blob/a46d2f84703c00e08936960d1746bac1964488df/tests/runtests.py#L31)). I fixed also this in #231.